docs/examples/kops-test-route53-subdomain.md

```diff
@@ -0,0 +1,870 @@
+  machineType: t2.micro
+  maxSize: 2
+  minSize: 2
+  role: Node
+  subnets:
+  - us-east-1a
+  - us-east-1b
+  - us-east-1c
+```
+
+Let's change minSize and maxSize to "3"
+
+```
+apiVersion: kops/v1alpha2
+kind: InstanceGroup
+metadata:
+  creationTimestamp: 2017-09-06T13:40:39Z
+  labels:
+    kops.k8s.io/cluster: mycluster01.kopsclustertest.kopeio.org
+  name: nodes
+spec:
+  image: kope.io/k8s-1.7-debian-jessie-amd64-hvm-ebs-2017-07-28
+  machineType: t2.micro
+  maxSize: 3
+  minSize: 3
+  role: Node
+  subnets:
+  - us-east-1a
+  - us-east-1b
+  - us-east-1c
+```
+
+Save it and review with `kops update cluster $NAME`:
+
+```bash
+kops update cluster $NAME
+``` 
+
+The last command will output:
+
+```bash
+I0906 10:16:30.619321   13607 executor.go:91] Tasks: 0 done / 75 total; 38 can run
+I0906 10:16:32.703865   13607 executor.go:91] Tasks: 38 done / 75 total; 14 can run
+I0906 10:16:33.592807   13607 executor.go:91] Tasks: 52 done / 75 total; 19 can run
+I0906 10:16:35.009432   13607 executor.go:91] Tasks: 71 done / 75 total; 4 can run
+I0906 10:16:35.320078   13607 executor.go:91] Tasks: 75 done / 75 total; 0 can run
+Will modify resources:
+  AutoscalingGroup/nodes.mycluster01.kopsclustertest.kopeio.org
+        MinSize                  2 -> 3
+        MaxSize                  2 -> 3
+
+Must specify --yes to apply changes
+``` 
+
+Now, let's apply the change:
+
+```bash
+kops update cluster $NAME --yes
+``` 
+
+Go for another coffee (or maybe a tee) and after some minutes check your cluster again with "kops validate cluster"
+
+```bash
+kops validate cluster
+
+Using cluster from kubectl context: mycluster01.kopsclustertest.kopeio.org
+
+Validating cluster mycluster01.kopsclustertest.kopeio.org
+
+INSTANCE GROUPS
+NAME                    ROLE    MACHINETYPE     MIN     MAX     SUBNETS
+master-us-east-1a       Master  t2.micro        1       1       us-east-1a
+master-us-east-1b       Master  t2.micro        1       1       us-east-1b
+master-us-east-1c       Master  t2.micro        1       1       us-east-1c
+nodes                   Node    t2.micro        3       3       us-east-1a,us-east-1b,us-east-1c
+
+NODE STATUS
+NAME                            ROLE    READY
+ip-172-20-103-68.ec2.internal   node    True
+ip-172-20-125-42.ec2.internal   master  True
+ip-172-20-33-58.ec2.internal    master  True
+ip-172-20-43-160.ec2.internal   node    True
+ip-172-20-64-116.ec2.internal   master  True
+ip-172-20-68-15.ec2.internal    node    True
+
+Your cluster mycluster01.kopsclustertest.kopeio.org is ready
+
+```
+
+You can see how your cluster scaled up to 3 nodes. 
+
+**SCALING RECOMMENDATIONS:**
+- Always think ahead. If you want to ensure to have the capability to scale-up to all available zones in the region, ensure to add them to the "--zones=" argument when using the "kops create cluster" command. Example: --zones=us-east-1a,us-east-1b,us-east-1c,us-east-1d,us-east-1e. That will make things simpler later.
```

Comment:
Maybe we should talk about having multiple instance groups, different node sizes, having them in one az to ensure ebs availabliltiy?

Reply:
You can always add regions and subnets as well. You cannot modify subnets once the ads in place.

Reply:
That can be expanded in a different chapter